pkg/release/uninstall.go

```diff
@@ -0,0 +1,81 @@
+package release
+
+import (
+	"errors"
+	"github.com/helmwave/helmwave/pkg/feature"
+	"github.com/helmwave/helmwave/pkg/helper"
+	"github.com/helmwave/helmwave/pkg/parallel"
+	log "github.com/sirupsen/logrus"
+	"helm.sh/helm/v3/pkg/action"
+	helm "helm.sh/helm/v3/pkg/cli"
+	"helm.sh/helm/v3/pkg/release"
+)
+
+func (rel *Config) Uninstall() (*release.UninstallReleaseResponse, error) {
+	cfg, err := helper.ActionCfg(rel.Options.Namespace, helm.New())
```

Comment:
TODO for future (probable should be an issue): there could be possible race with running helm commands with different namespaces. So we can make an RWMutex for it (maybe there are some fancy wrappers for this)